blocks/hero/index.js

```diff
@@ -0,0 +1,230 @@
+								</a>
+							</div>
+						</div>
+				</section>
+			);
+		},
+		save( { attributes } ) {
+
+			/* @todo Clicking this should direct to the search results template (doesn't exist yet). */
+			/* @todo Make it work with the CPT categories which currently don't exist yet. */
+
+			const ampValueProp = {
+				'[value]': 'fields_query'
+			};
+
+			// @todo Confirm possible solution for replacing [src] -- add data-ampsrc instead and replace it with filters when displaying the post.
+			const ampSrcProp = {
+				'data-ampsrc': "'" + travelGlobals.apiUrl + "wp/v2/categories?search=' + fields_query_live"
+			};
+
+			const departureClassNameProp = {
+				'[class]': "'travel-date-input relative bold flex-auto' + (fields_departure ? ' travel-date-input-touched' : '')"
+			};
+			const returnClassNameProp = {
+				'[class]': "'travel-date-input relative bold flex-auto' + (fields_return ? ' travel-date-input-touched' : '')"
+			};
+			const ampDisabledProp = {
+				'[disabled]': '!fields_departure'
+			};
+
+			return (
+				<section className='travel-hero relative'>
+					<div className='travel-hero-content max-width-3 mx-auto absolute top-0 left-0 right-0 flex self-end items-center'>
+						<div className='travel-hero-content-inner relative px1 md-px2 flex-auto self-end'>
+							<header>
+								<h1 className='travel-hero-heading mb2 line-height-1'>{ attributes.heading }</h1>
+								<h2 className='travel-hero-subheading line-height-2 bold xs-hide sm-hide'>{ attributes.subheading }</h2>
+							</header>
+							<div className='travel-hero-search'>
+								<label className='travel-input-icon travel-shadow flex col-12 relative rounded'>
+									<input className='travel-input travel-input-big travel-input-clear border block col-12 rounded' list='locations' type='text' name='query' placeholder='Where would you like to go?' on='
+						              change:AMP.setState({
+						                fields_query: event.value,
+						                fields_query_live: event.value,
+						                fields_query_edited: query_query != event.value
+						              });
+						              input-debounced:AMP.setState({
+						                fields_query_live: event.value
+						              });
+						            ' value='' { ...ampValueProp } />
+									<svg className='travel-icon' viewBox='0 0 74 100'><path fill='currentColor' d='M40.18 95.404A3.944 3.944 0 0 1 37 97a3.944 3.944 0 0 1-3.18-1.596C28.268 87.787 5 54.66 5 34.334 5 17.027 19.327 3 37 3c17.673 0 32 14.028 32 31.333 0 20.327-23.267 53.454-28.82 61.07zM37 14.75c-11.046 0-20 8.768-20 19.583 0 10.816 8.954 19.584 20 19.584s20-8.768 20-19.584c0-5.193-2.107-10.174-5.858-13.847-3.75-3.672-8.838-5.736-14.142-5.736z'></path></svg>
+								</label>
+
+								<amp-list layout='fixed-height' height="80px" src={travelGlobals.apiUrl + 'wp/v2/categories'} {...ampSrcProp } aria-live='polite'>
+									<template type='amp-mustache'>
```

Comment:
Since the mustache template suggests categories (locations) but we are using the default WP search, should we remove the template for now?